app/models/project_submission.rb

```diff
@@ -1,4 +1,6 @@
 class ProjectSubmission < ApplicationRecord
```

Comment:
I think we should add a default scope to the ProjectSubmissions model so all the discarded submissions are filtered out by default, that will allow us to remove the `discarded_at: nil` conditions on the viewable scope and in the lessons_helper.

Discard have an example of using a default scope here: https://github.com/jhawthorn/discard#default-scope

Reply:
Much better this way, thanks! I just kept it simple with `default_scope -> { kept }` in the submission model. I think that's good enough? Seems to be working the same. Now, if we want to find a discarded post, I guess we can do `ProjectSubmission.with_discarded`.

Reply:
Perfect 👌